web/src/app/rotations/RotationDeleteDialog.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react'
-import { useDispatch } from 'react-redux'
 import p from 'prop-types'
 import gql from 'graphql-tag'
 import Spinner from '../loading/components/Spinner'
 import FormDialog from '../dialogs/FormDialog'
 import { useQuery, useMutation } from 'react-apollo'
-import { push } from 'connected-react-router'
+import { get } from 'lodash-es'
```

Comment:
Do you think if we can use `get` from `lodash` instead of `lodash-es` here?  Related to https://github.com/target/goalert/issues/101 where we witnessed slowness with the `lodash-es` imports  -- thoughts?

Maybe this doesn't matter any more (so I should delete the issue)?

Reply:
We should only use one or the other for now, so we don't embed both libraries in our build. Though importing `_` and using `_.get` might be more clear.